ttnn/cpp/ttnn/async_runtime.cpp

```diff
@@ -115,12 +115,7 @@ void queue_synchronize(CommandQueue& cq) {
     Finish(cq);
 }
 
-void event_synchronize(Device* device, std::shared_ptr<Event> event) {
-    device->push_work([event] () {
-        EventSynchronize(event);
-    });
-    device->synchronize();
-}
+void event_synchronize(Device* device, std::shared_ptr<Event> event) { EventSynchronize(event); }
```

Comment:
Hi @tt-asaigal 
Could you remove `Device* device` that is usused?

Reply:
Hey, this has been updated. Thanks!